cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json

```diff
@@ -527,7 +527,20 @@
         },
         {
           "widget-type": "keyvalue",
-          "label": "Labels",
+          "label": "Common Labels",
+          "name": "labels",
+          "description": "Specifies labels for the all Dataproc entities.",
+          "widget-attributes": {
+            "showDelimiter": "false",
+            "delimiter": ",",
```

Comment:
I would be consistent between clusterLabels and labels for separators. I don't know the history of why we use the different separator, but it makes sense to make it easy to move values between this two fields

Reply:
Changed. Now clusterLabels and labels are consistent but if users are setting the same labels property in cdap-site and provisioner, they will need to use different delimiters.